src/vdaf.rs

```diff
@@ -37,7 +38,7 @@ pub enum VdafError {
 }
 
 /// An additive share of a vector of field elements.
-#[derive(Clone, Debug, Serialize, Deserialize)]
+#[derive(Clone, Debug, PartialEq, Serialize, Deserialize)]
```

Comment:
Do you think we should take `Serialize, Deserialize` off of this and some other structs in the crate? On the one hand, we want clients to use the custom codec methods you've added, but on the other hand, it could be useful to store these messages in arbitrary formats. For instance we already store Priov2 test vectors as JSON, but we might be able to restrict the `serde` dependency to just the test targets, which could make the production config of the crate quite a bit smaller.

Reply:
I don't see any reason to allow custom serialization of the messages. Test vectors for VDAFs will just be the messages encoded in hex, e.g.: https://www.ietf.org/archive/id/draft-irtf-cfrg-hpke-12.html#name-test-vectors

Reply:
Yes, you need `PartialEq` to implement `Eq`. I'm saying we should derive both.

Let's remove `Serialize, Deserialize`. I agree that there's no need for custom serialization.

Reply:
Done and done!